src/mgr/OSDPerfMetricReport.h

```diff
@@ -0,0 +1,29 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#ifndef OSD_PERF_METRIC_REPORT_H_
+#define OSD_PERF_METRIC_REPORT_H_
+#include "include/denc.h"
+//#include "mgr/OSDPerfMetricQuery"
+
+typedef int OSDPerfMetricQueryID;  //Temporary; to be moved in mgr/OSDPerfMetricQuery.h ?
+
+struct OSDPerfMetricReport
+{
+  OSDPerfMetricQueryID query_id; 
```

Comment:
Just note, that this query_id is not what is returned to the caller of `OSDPerfMetricCollector::add_query`. Because we may have many the same queries added (with different ID). So we would need either to keep yet another list of IDs (a mapping between query and its ID) and send the ID together with the query in MMgrConfigure, or use the query as a key  (like I do in `OSDPerfMetricCollector`), that is send OSDPerfMetricQuery instead of OSDPerfMetricQueryID in OSDPerfMetricReport.

Reply:
Okay, thanks for the feedback!